static/templates/composer.tpl

```diff
@@ -1,4 +1,4 @@
-<div component="composer" class="composer<!-- IF resizable --> resizable<!-- ENDIF resizable -->">
+<div component="composer" class="composer<!-- IF resizable --> resizable<!-- ENDIF resizable --><!-- IF isTopicOrMain --> topic-or-main<!-- ENDIF isTopicOrMain -->">
```

Comment:
Remove and re-submit as separate PR please. There are two composer templates, and you have to ensure that `isTopicOrMain` is passed to the `/compose` route as well.